osu.Framework/Graphics/Containers/CustomizableTextContainer.cs

```diff
@@ -88,9 +87,10 @@ public int AddPlaceholder(Drawable drawable)
         /// <param name="name">The name of the placeholder.</param>
         /// <param name="iconFactory">The icon factory matching <paramref name="name"/>, if the method returned <see langword="true"/>.</param>
         /// <returns>Whether an icon factory was found for the given <paramref name="name"/>.</returns>
-        internal bool TryGetIconFactory(string name, out Delegate iconFactory) => iconFactories.TryGetValue(name, out iconFactory);
+        internal bool TryGetIconFactory(string name, [MaybeNullWhen(false)] out Delegate iconFactory) => iconFactories.TryGetValue(name, out iconFactory);
```

Comment:
Prefer the other way around if I'm honest, i.e.

```suggestion
        internal bool TryGetIconFactory(string name, [NotNullWhen(true)] out Delegate? iconFactory) => iconFactories.TryGetValue(name, out iconFactory);
```